app/pods/application/route.js

```diff
@@ -36,6 +37,14 @@ export default class ApplicationRoute extends Route {
     if (this.session.isAuthenticated && !this.currentSession.hasValidGroup) {
       this.transitionTo('accountless-users');
     }
+
+    await this.store.query('meeting-kind', {
+      'page[size]': PAGE_SIZE.MEETING_KIND,
+      filter: {
+        ':has:position': true, // There is an ext:MinisterraadType resource called Annex which we don't want to show in dropdowns. It does not have a schema:position property, so we use that fact to filter it out
```

Comment:
`:has:` filter flag is enkel bedoeld voor relaties, niet voor attributes. Zoals al aangegeven in backend PR kan hier beter gefilterd worden op `:has-no:narrower`.